Sources/TuistKit/Commands/InstallCommand.swift

```diff
@@ -27,10 +27,18 @@ public struct InstallCommand: AsyncParsableCommand {
     )
     var update: Bool = false
 
+    @Flag(
+        name: .shortAndLong,
+        help: "Look up dependencies in the swift package registry and use the registry to retrieve them instead of source control when possible.",
```

Comment:
```suggestion
        help: "Look up dependencies in the Swift Package Registry and use the registry to retrieve them instead of source control when possible.",
```